collect_app/src/main/java/org/odk/collect/android/activities/viewmodels/FormDownloadListViewModel.java

```diff
@@ -207,4 +226,37 @@ public boolean wasLoadingCanceled() {
     public void setLoadingCanceled(boolean loadingCanceled) {
         this.loadingCanceled = loadingCanceled;
     }
+
+    public void logDownloadAnalyticsEvent(int downloadedFormCount, String serverUrl) {
+        String analyticsEvent = getDownloadAnalyticsEvent(downloadedFormCount);
+        String analyticsDesc = getDownloadAnalyticsDescription(serverUrl);
+        analytics.logEvent(analyticsEvent, analyticsDesc);
+    }
+
+    String getDownloadAnalyticsEvent(int downloadedFormCount) {
```

Comment:
These methods (`getDownloadAnalyticsEvent` and `getDownloadAnalyticsDescription`) feel like they should be private now as they're just used as helpers for `logDownloadAnalyticsEvents` and don't feel like they should be part of the external interface of the object. That would also mean reworking the tests a bit.